package.json

```diff
@@ -113,6 +113,8 @@
     "react-test-renderer": "^18.0.0",
     "resolution-independence": "^1.0.0",
     "resolve": "^1.22.0",
+    "sass": "^1.49.0",
```

Comment:
Is this necessary?

Reply:
Without this module, errors occurred.

> Failed to compile.
Cannot find module 'sass'
Require stack:
- /home/taeyoung.hong/webos_build_server/WRO-630/cli2/node_modules/sass-loader/dist/utils.js
- /home/taeyoung.hong/webos_build_server/WRO-630/cli2/node_modules/sass-loader/dist/index.js
- /home/taeyoung.hong/webos_build_server/WRO-630/cli2/node_modules/sass-loader/dist/cjs.js
- /home/taeyoung.hong/webos_build_server/WRO-630/cli2/node_modules/loader-runner/lib/loadLoader.js
- /home/taeyoung.hong/webos_build_server/WRO-630/cli2/node_modules/loader-runner/lib/LoaderRunner.js
- /home/taeyoung.hong/webos_build_server/WRO-630/cli2/node_modules/webpack/lib/NormalModuleFactory.js
- /home/taeyoung.hong/webos_build_server/WRO-630/cli2/node_modules/webpack/lib/Compiler.js
- /home/taeyoung.hong/webos_build_server/WRO-630/cli2/node_modules/webpack/lib/webpack.js
- /home/taeyoung.hong/webos_build_server/WRO-630/cli2/node_modules/webpack/lib/index.js
- /home/taeyoung.hong/webos_build_server/WRO-630/cli2/commands/serve.js
- /home/taeyoung.hong/webos_build_server/WRO-630/cli2/bin/enact.js

Reply:
Have you installed `sass` from your app? Please recheck.